CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribution Guidelines
+
+This document aims to provide guidelines for merging changesets into the sparrow master branch.
+
+For the purpose of this document, we classify changesets/pull requests as:
+
+* big: major changes to core code
+* non-trivial: significant changes, fixes and improvements. Especially if they change core modules.
```

Comment:
non-trivial: significant changes, fixes and improvements - especially changes to the core modules.